frontend/src/bundles/auth/store/slice.ts

```diff
@@ -19,7 +20,16 @@ const initialState: State = {
 const { reducer, actions, name } = createSlice({
     initialState,
     name: 'auth',
-    reducers: {},
+    reducers: {
+        setUser: (
+            state,
+            action: PayloadAction<UserWithProfileRelationAndOauthConnections>,
```

Comment:
this type `UserWithProfileRelationAndOauthConnections`, should be non-actual after you resolve conflicts

Why did you add setUser?